net-misc/anydesk/anydesk-6.2.1.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit desktop optfeature systemd xdg-utils
+
+DESCRIPTION="Feature rich multi-platform remote desktop application"
+HOMEPAGE="https://anydesk.com"
+SRC_URI="https://download.anydesk.com/linux/${P}-amd64.tar.gz
+	https://download.anydesk.com/linux/generic-linux/${P}-amd64.tar.gz"
```

Comment:
Why are there 2 links here with the same name?  The second one returns 404.  Is this some peculiarity about the way Anydesk release their Linux client?

Reply:
Hmm, good spot.

At some point they changed the way the structured their download repository such that old stuff was moved into the generic folder.  Or at least a significant portion of the files.  The folder still exist, and at the time it looked like one of the two URLs should keep working.  I'm OK with only listing the first, it seems they keep at least from version 6.0.1 onwards still there.

I just re-looked at the x86 situation as well, whilst there are no .tar.gz builds available, it does look like there are rpm builds available, I have no interest in that, but if someone needs this on x86 still (I consider this to be unlikely) then we can certainly look at that.

This double download URL was first included in version 5.5.0 which I added due to the move of 5.5.0 into the generic folder, newest versions are not present in the generic folder, meaning that over time the URL changes (in a predictable manner).